### PR TITLE
update locales/zh-CN

### PR DESCRIPTION
**Pull request by hooklife**

### CODE REVIEW

**Comment by kazup01** [comment]
Thanks @hooklife for your contribution.
Is there anyone who can review it in Chinese?

**Comment by ChenLiZhan** [comment]
@kazup01 
I am glad to review for the Chinese part!

**Comment by kazup01** [comment]
Thanks @ChenLiZhan !

**Comment by kazup01** [comment]
Hi @hooklife , thanks for your contribution! Could you fix conflict?

File: locales/zh-CN.json [hunk 1]
```diff
@@ -146,8 +149,62 @@
...
+	"Hide Help": "隐藏帮助",
+	"wordpress": "Wordpress",
+	"Add Storage": "添加存储",
+	"Name": "名称",
+	"Type": "类型",
+	"File System": "文件",
+	"Setting up 3rd-party cloud storage integration:": "设置整合第三方云存储:",
+	"Cloud-Syncing-and-Backup": "云端同步和备份",
+	"Location": "路径",
+	"Select Folder": "选择目录",
```
Comment (ChenLiZhan) [inline locales/zh-CN.json]: [IMO] Does it seem to be better if using `选择文件夹` to differentiate with [line 199](https://github.com/BoostIO/Boostnote/pull/1928/files#diff-d42b9883d736cfed006cb4ff0cd44ed0R199)?

File: locales/zh-CN.json [hunk 2]
```diff
@@ -146,8 +149,62 @@
...
+	"Add": "添加",
+	"Available Keys": "可用按键",
+	"Select Directory": "选择目录",
+	"copy": "副本",
+	"Create new folder": "创建新文件夹",
+	"Folder name": "文件夹名称",
+	"Create": "创建",
+	"Unlink Storage": "取消链接",
```
Comment (ChenLiZhan) [inline locales/zh-CN.json]: [IMO] `"取消存储链接"` looks more specific to me.

File: locales/zh-CN.json [hunk 3]
```diff
@@ -146,8 +149,62 @@
...
+	"Delete Folder": "删除文件夹",
+	"Select directory": "选择目录",
+	"Select a folder to export the files to": "选择一个导出目录",
+	"Description...": "描述...",
+	"Publish Failed": "发布失败",
+	"Check and update your blog setting and try again.": "检查并修改你的博客设置后重试。",
+	"Delete a snippet": "删除一个代码片段",
+	"This work cannot be undone.": "此操作无法撤销。",
+	"Help": "帮助",
+	"Hungarian": "Hungarian",
```
Comment (ChenLiZhan) [inline locales/zh-CN.json]: [Q] I don't know where the word `"Hungarian"` used actually. Does it mean `匈牙利语` here?

**Comment by hooklife** [comment]
@ChenLiZhan 
thanks a lot,I've fixed it

**Comment by Rokt33r** [comment]
@hooklife @ChenLiZhan Thanks for the contributing 💯 💯 💯

**Pull request merged**